### PR TITLE
Draft: Remove config-server dependencies for older Arkcase versions

### PR DESCRIPTION
**Pull request by jovestefanovski**
This is initially created for MDSAP and it can be used in future where we have to deploy older Arkcase versions through the installer.

### CODE REVIEW

File: vagrant/provisioning/roles/start-arkcase/tasks/main.yml [hunk 1]
```diff
@@ -4,19 +4,6 @@
     path: "{{ root_folder }}/common/symmetricKey.decrypted"
     state: absent
 
-- name: start config server
-  become: yes
-  systemd:
-    daemon_reload: true
-    name: "config-server"
-    state: started
```
Comment (bojmil-armedia) [inline vagrant/provisioning/roles/start-arkcase/tasks/main.yml:12]: Are we starting it somewhere else? Why don't you just put a when condition on arkcase_version?

Comment (bojmil-armedia) [reply vagrant/provisioning/roles/start-arkcase/tasks/main.yml:12]: It should not run prior arkcase version 3.3.1

**Comment by david-oc-miller** [comment]
@jovestefanovski , @dzdimov , do we still need remove config-server dependencies for older ArkCase versions?  It's hard to see the actual change now since some of the Alfresco 7 / PostgreSQL changes are layered on top.

**Comment by jovestefanovski** [comment]
This PR was specifically for MDSAP with PR source DSO-480. And this shouldn't be merged into develop.
With just using this code with single run we were able to deploy older ArkCase version with PostgreSQL to suit MDSAP.

**Comment by david-oc-miller** [comment]
Thanks @jovestefanovski ... so we can close this MR without merging?

**Pull request closed without merging**